example_spec.rb

```diff
@@ -16,3 +16,8 @@
 caller.first
 caller_locations[1]
 caller_locations.first
+##Patterns: Layout_ArrayAlignment
+
+menu_item = ['Spam', 'Spam',
+##Info: Layout_ArrayAlignment
+'Spam', 'Hello']
```

Comment:
![Codacy](https://app.staging.codacy.org/assets/images/favicon.png) Issue found: [Align the elements of an array literal if they span more than one line.](https://app.staging.codacy.org/manual/josemiguelmelo/example-rubocop/pullRequest?prid=9434)